src/Features/Core/Portable/ExternalAccess/Pythia/Api/PythiaCompletionProviderBase.cs

```diff
@@ -42,7 +43,7 @@ public static CompletionItem CreateSymbolCompletionItem(
         SupportedPlatformData? supportedPlatforms = null,
         ImmutableDictionary<string, string>? properties = null,
         ImmutableArray<string> tags = default)
-        => SymbolCompletionItem.CreateWithSymbolId(displayText, displayTextSuffix: null, symbols, rules, contextPosition, sortText, insertionText,
+        => SymbolCompletionItem.CreateWithSymbolId(displayText, displayTextSuffix: null, symbols.ToImmutableArray(), rules, contextPosition, sortText, insertionText,
```

Comment:
>ToImmutableArray [](http://example.com/codeflow?start=97&length=16)

This is potentially an allocation where it wasn't before, but only if they weren't using an ImmutableArray and it's limited to Pythia.